File: problemas/1100/1101.py
while True:
    entrada = list(map(int, input().split(' ')))
    menor = min(entrada[0], entrada[1])
    maior = max(entrada[0], entrada[1])

    if menor <= 0 or maior <= 0:
        break

    soma = 0

    for i in range(menor, maior + 1):
        print(f'{i} ', end='')
        soma += i

    print(f'Sum={soma}')

File: problemas/1100/1101.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

while (true) {
  const [m, n] = entrada.shift().split(' ');

  if (n <= 0 || m <= 0) {
    break;
  }

  const menor = Math.min(m, n);
  const maior = Math.max(m, n);
  let soma = 0;

  for (let i = menor; i <= maior; i++) {
    process.stdout.write(i + ' ');
    soma += i;
  }

  console.log(`Sum=${soma}`);
}